spec/system/management/budget_investments_spec.rb

```diff
@@ -98,6 +98,17 @@
         expect(page).not_to have_content "Plant trees"
       end
     end
+
+    scenario "when user has not been selected we can't create a budget investment" do
+      Setting["feature.user.skip_verification"] = "true"
+      login_managed_user(User.new)
```

Comment:
It certainly looks strange to add a `login_managed_user` line in a test where we check no user is logged in :thinking:.

A few alternatives:

1. Add `context "when a user has been selected"`, include all existing tests there, and move this test to another `context "when no user has been selected"`
2. Remove `before { login_managed_user(user) }` from the top of the file and add it to each test
3. Replace this line with `logout_managed_user` and create that method (maybe with `allow_any_instance_of(Management::BaseController).to receive(:managed_user).and_call_original`

I think I prefer option 1 :thinking:, but haven't thought enough about it.

The same applies to the other tests in this pull request having this line.